### PR TITLE
Fixes #11143: Update screenshot for interface usage page 

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/11143

### CODE REVIEW

File: 20_usage/10_web_interface.txt [hunk 1]
```diff
@@ -23,30 +23,67 @@ image::./images/rudder-home.png[Home menu]
...
+
+image::./images/technique_editor/1-rudder-technique-editor.png[Technique editor]
+
+.Technique details
+
+image::./images/technique_editor/5-configure-generic-method.png[Technique details]
+
+==== Settings
+
+The Settings section provides modify Rudder setup: you can setup the
```
Comment (peckpeck) [inline 20_usage/10_web_interface.txt]: provides modify ?

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by peckpeck** [comment]
I still see the error

File: 20_usage/10_web_interface.txt (hunk 1 above)
Comment (VinceMacBuche) [inline 20_usage/10_web_interface.txt]: ha...

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
I had'nt seen your feedback and correct an error in tests ;)

File: 20_usage/10_web_interface.txt [hunk 2]
```diff
@@ -23,30 +23,68 @@ image::./images/rudder-home.png[Home menu]
 
 ==== Node Management
 
-In the Node Management section, you will find the validation tool for new
+In the Node Management section, you will find, the list of all Nodes, the validation tool for new
```
Comment (ncharles) [inline 20_usage/10_web_interface.txt]: i think the first added comma is too much, it should read
In the Node Management section, you will find the list of all Nodes, the validation

File: 20_usage/10_web_interface.txt [hunk 3]
```diff
@@ -23,30 +23,68 @@ image::./images/rudder-home.png[Home menu]
...
-.Configuration Policy (Rules) screen
+image::./images/Directive_management.png[Directives]
 
-image::./images/rudder-configurationpolicy-rules.png[Configuration Policy]
+==== Utilities
 
-==== Administration
+This section contains tools useful for your everyday usage of Rudder.
+This is where you will find the technique editor, the event logs table
+or the change requests if you have enable that feature.
```
Comment (ncharles) [inline 20_usage/10_web_interface.txt]: if you have enabled

File: 20_usage/10_web_interface.txt [hunk 4]
```diff
@@ -23,30 +23,68 @@ image::./images/rudder-home.png[Home menu]
...
+
+.Technique details
+
+image::./images/technique_editor/5-configure-generic-method.png[Technique details]
+
+==== Settings
+
+The Settings section provides you a way to modify your Rudder setup: you can setup the
+available networks for the Policy Server, configure agent run and policy mode,
+enable webapp options and manage installed plugins.
```
Comment (ncharles) [inline 20_usage/10_web_interface.txt]: enable Web Interface options

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**